### PR TITLE
Add the foundation of the HTML block.

### PR DESCRIPTION
**Pull request by mbaechtold**
This adds a simple block which can be added to a content page.

No HTML can be added to the block right now, the block simply renders some static content for now.

### CODE REVIEW

File: ftw/htmlblock/profiles/default/types/ftw.htmlblock.HtmlBlock.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<object name="ftw.htmlblock.HtmlBLock"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="ftw.htmlblock">
+
+    <!-- Basic metadata -->
+    <property name="title" i18n:translate="">HtmlBlock</property>
```
Comment (jone) [inline ftw/htmlblock/profiles/default/types/ftw.htmlblock.HtmlBlock.xml]: I'd prefer `HTML Block` here. Any objections?

Comment (mbaechtold) [inline ftw/htmlblock/profiles/default/types/ftw.htmlblock.HtmlBlock.xml]: Yeah, I already planned to use `HTML block` everywhere (`HTML-Block` in German translation).

File: ftw/htmlblock/profiles/default/types/ftw.htmlblock.HtmlBlock.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    <!-- enabled behaviors -->
+    <property name="behaviors">
+        <element value="plone.app.content.interfaces.INameFromTitle" />
+        <element value="ftw.simplelayout.interfaces.ISimplelayoutBlock" />
+        <element value="plone.app.referenceablebehavior.referenceable.IReferenceable"/>
+    </property>
+
+    <!-- View information -->
+    <property name="default_view">folder_contents</property>
```
Comment (jone) [inline ftw/htmlblock/profiles/default/types/ftw.htmlblock.HtmlBlock.xml]: this seems wrong

File: ftw/htmlblock/profiles/default/types/ftw.htmlblock.HtmlBlock.xml (hunk 1 above)
Comment (jone) [inline ftw/htmlblock/profiles/default/types/ftw.htmlblock.HtmlBlock.xml]: 👍

**Comment by jone** [comment]
👍 you may merge whenever you want 😉

**Comment by mbaechtold** [comment]
Thanks for the review, @jone :tada:

**Pull request merged**